### ISSUE TITLE
Directions button and links no longer work

### ISSUE THREAD

**Issue by sconim** [Divested-Mobile/Maps#18]
Hi,

The **Directions button** (following a search) no longer works. When pressed, nothing happens. Just to clarity, the Direction button on the main page (screenshot 1) still works. The one that no longer works is shown in screenshot 2. 
![Screenshot_20241206_111030_GMaps WV](https://github.com/user-attachments/assets/4890812f-6d58-4abf-968c-aa8c7bfb5135)
![Screenshot_20241206_113155_GMaps WV](https://github.com/user-attachments/assets/a90c78ec-6db6-4040-a570-2d296893f308)

Also, the Google provided **links** to webpages (following a search - where available) no longer works.

GMaps 1.12
Android 13, microG 3.4, Samsung S20

**Comment by trinity-geology-unstable** [comment]
Hi, I am also facing this bug.

**Comment by sconim** [comment]
Is this app still maintained? I submitted the bug 3 weeks ago and there is no feedback whatsoever.

**Comment by SkewedZeppelin** [comment]
Yes it is still maintained, I just did a release with updated translations.
But there is only me, it is still on my TODO list to fix.

**Comment by sconim** [comment]
I've been relying heavily on your app for quite a few years now, and fixing the few remaining bugs mentioned here would mean a lot to us. Thank you for the reply and the great app.
